main.go

```diff
@@ -0,0 +1,300 @@
+				Usage:   "Address of the bidder",
+				EnvVars: []string{"BIDDER_ADDRESS"},
+				Value:   "mev-commit-bidder:13524",
+			},
+			&cli.BoolFlag{
+				Name:    "use-payload",
+				Usage:   "Use payload for transactions",
+				EnvVars: []string{"USE_PAYLOAD"},
+				Value:   true,
+			},
+			&cli.StringFlag{
+				Name:     "rpc-endpoint",
+				Usage:    "RPC endpoint when use-payload is false",
+				EnvVars:  []string{"RPC_ENDPOINT"},
+				Required: false,
+			},
+			&cli.StringFlag{
+				Name:     "ws-endpoint",
+				Usage:    "WebSocket endpoint for transactions",
+				EnvVars:  []string{"WS_ENDPOINT"},
+				Required: true,
+			},
+			&cli.StringFlag{
+				Name:      "private-key",
+				Usage:     "Private key for signing transactions",
+				EnvVars:   []string{"PRIVATE_KEY"},
+				Required:  true,
+				Hidden:    true,
+				TakesFile: false,
+			},
+			&cli.Uint64Flag{
+				Name:    "offset",
+				Usage:   "Offset value for transactions",
+				EnvVars: []string{"OFFSET"},
+				Value:   1,
+			},
+			&cli.Float64Flag{
+				Name:    "bid-amount",
+				Usage:   "Amount to bid",
+				EnvVars: []string{"BID_AMOUNT"},
+				Value:   0.001,
+			},
+			&cli.Float64Flag{
+				Name:    "bid-amount-std-dev-percentage",
+				Usage:   "Standard deviation percentage for bid amount",
+				EnvVars: []string{"BID_AMOUNT_STD_DEV_PERCENTAGE"},
+				Value:   100.0,
+			},
+			&cli.UintFlag{
+				Name:    "num-blob",
+				Usage:   "Number of blobs to send (0 for ETH transfer)",
+				EnvVars: []string{"NUM_BLOB"},
+				Value:   0,
+			},
+			&cli.UintFlag{
+				Name:    "default-timeout",
+				Usage:   "Default timeout in seconds",
+				EnvVars: []string{"DEFAULT_TIMEOUT"},
+				Value:   15, // Default to 15 seconds
+			},
+		},
+		Action: func(c *cli.Context) error {
+			// Retrieve flag values
+			bidderAddress := c.String("bidder-address")
+			usePayload := c.Bool("use-payload")
+			rpcEndpoint := c.String("rpc-endpoint")
+			wsEndpoint := c.String("ws-endpoint")
+			privateKeyHex := c.String("private-key")
+			offset := c.Uint64("offset")
+			bidAmount := c.Float64("bid-amount")
+			stdDevPercentage := c.Float64("bid-amount-std-dev-percentage")
+			numBlob := c.Uint64("num-blob")
+			defaultTimeoutSeconds := c.Uint("default-timeout") // New variable
+			defaultTimeout := time.Duration(defaultTimeoutSeconds) * time.Second
+
+			// Log the defaultTimeout value
+			log.Info().
+				Str("bidderAddress", bidderAddress).
+				Str("rpcEndpoint", bb.MaskEndpoint(rpcEndpoint)).
+				Str("wsEndpoint", bb.MaskEndpoint(wsEndpoint)).
+				Uint64("offset", offset).
+				Bool("usePayload", usePayload).
+				Float64("bidAmount", bidAmount).
+				Float64("stdDevPercentage", stdDevPercentage).
+				Uint64("numBlob", numBlob).
+				Bool("privateKeyProvided", privateKeyHex != "").
+				Uint("defaultTimeoutSeconds", defaultTimeoutSeconds).
+				Msg("Configuration values")
+
+			cfg := bb.BidderConfig{
+				ServerAddress: bidderAddress,
+				LogFmt:        "json",
```

Comment:
What are these `Log*` files for when thy are set but not used?

Reply:
removed